### DISCUSSION TITLE
2020年 | 年终总结

### DISCUSSION THREAD

**Discussion by fuzhengwei** [fuzhengwei/CodeGuide#194] [Show and tell]
## 写下你的2020总结
1. 这一年你完成年初的目标了吗？
2. 这一年你跑步健身锻炼了吗？
3. 这一年你有多少技术成长？
4. 这一年你和家人团聚过几次？
5. 这一年你在公司升职加薪了吗？
6. 这一年你有一辆自己的车了吗？
7. 这一年你学会理财了吗？
8. 这一年你还有没完成的事情吗？
9. 这一年你和她在一起了吗？
10. 这一年你对`自己`还好吗？

**Comment by fuzhengwei** [comment]
`快到年底了，写个总结吧！`

- 关注我的粉丝朋友，谢谢你！滴水之恩，永不相忘！ 
- 我没照顾到的伙伴，对不起！我不是有意忽略了你。 
- 致我相识的每一位同好，所求皆如愿、所行化坦途。 
- 再见2020，迎接2021！

**目标&结果**
1. 个人：身体还算健康，也跑步了，但后来太冷了，只能早上起来做做俯卧撑。
2. 公司：顺利答辩晋升到架构师岗，还需继续努力
3. 家庭：好在离家近，几乎周末只要不加班，都可以回家
4. 成长：感觉有那么一点点技术成长，但还需继续挖掘深度
5. 其他：做了一个公众号：bugstack虫洞栈，粉丝从1299涨到1.7万，写了一本PDF《重学Java设计模式》，全网下载量15万+

**Reply by LGBoy** [reply]
反对

**Reply by Shing20** [reply]
反对无效！

**Comment by mifengjun** [comment]
#### 2020
2020 年收获很多。主要可能是因为接触到了很多爱学习的小伙伴吧，这一切都归功于傅哥。这一年过的充实而快乐，几乎全年都在学习的状态，当然啊，我坦白，也玩了一阵子剑网三。哈哈哈，通过这一年对自己技术的复盘，发现我有着我自己都害怕的理解能力。我特担心哪天这状态没有了，所以这一年就不停的学不停的总结。

#### 2021
2021年嘛，有一丝胆怯，同样也有一腔热血。目标就是希望自己的技术能够到达一个符合目前自己状态的标准。除了自身以外嘛，还有好多目标，就立前面一个 flag 好了。我怕立多了我又乱了。让我们一起期待 2021 年的总结吧！

**Reply by fuzhengwei** [reply]
加油搞起！2021年年末，再回来，回复！

**Comment by Shing20** [comment]
目标&结果

个人：身体还算健康，基本每周跑步5-6KM，近期每天俯卧撑30*2。
公司：无升职加薪(业务情况也不好)，还需继续努力，正考虑从外部突破！
家庭：爷爷，奶奶今年都生过大病，也有其它亲人去世；父母身体还没有检查；小孩成长计划还没有做完；
成长：技术进步不大，但已经有规划的进行；个人学历还会继续提升，这是态度问题（有与没有，想与不想）；人际关系处理基本原则升级完成；还是要少在群里bb
其他：今年下半年至今已经记录49篇技术读书笔记，小傅哥《面经系列》阅读中。“慢慢来，成长最快”让目标可执行，可量化，以结果为导向，以收入与核心争力为中心。只能相信未来，相信奋斗，哪怕高标准目标只实现了60%！

**Reply by fuzhengwei** [reply]
赞！
